lib/node/runner/checker_ballot_transaction.go

```diff
@@ -195,16 +197,36 @@ func ValidateOp(st *storage.LevelDBBackend, op transaction.Operation) (err error
 			err = errors.ErrorBlockAccountAlreadyExists
 			return
 		}
+		// If it's a frozen account we check that only whole units are frozen
+		if casted.Linked != "" && (casted.Amount%common.Unit) != 0 {
```

Comment:
if the casted.Amout is 0 then the result of the calculation also 0.
where validate the amount?

Reply:
In [`IsWellFormed`](https://github.com/bosnet/sebak/blob/f0067187f6191b1232f6344834bcc06e50b63bda/lib/transaction/operation_create_account.go#L28-L31)